docs/src/administration/users.md

```diff
@@ -1,138 +1,162 @@
+
+* `platform user:delete`
 
-After adding `alice@example.com`, Alice will receive an invitation email asking to confirm her details and optionally, register for a Platform.sh account.
+  ```bash
+  platform user:delete user1@example.com
+  ```
 
-To give Bob different levels of access depending on the environment type, in the current project:
- 
--  **Viewer** role to the **Production environment**
--  **Contributor** role to **all Staging type environments**
--  **Admin** role to **all Development type environments**
+  Use `platform list` to get the full list of commands.
 
-You would run:
+  After you delete a user from a project or an environment type, you must trigger a redeploy to propagate SSH access changes to each environment. You can redeploy by using the CLI command platform `redeploy`.
 
-```bash
-platform user:role bob@example.com -r production:viewer -r staging:contributor -r development:admin
-```
+{{< /codetabs >}}
 
-Use `platform list` to get the full list of commands.
+Once you delete a user, they won't be able to access the project anymore.
 
-## User access and integrations
 
-If you have setup an [external integration](/integrations/source/_index.md) to GitHub, GitLab, or Bitbucket, this adds an additional layer of access control to the project that you will need to be aware of.
+### Change user permissions from the console
 
-For example, if you invite a user with **Project Admin** role to a project on Platform.sh, but you haven't invited them to the remote repository on GitHub / GitLab, they might be [unable to clone the project](/administration/web/_index.md#git) locally.
+To change the user permissions, follow the steps:
 
-In this example, using either `platform get` with the CLI:
+{{< codetabs >}}
+---
+title="In the console"
+---
+1. Go to your console and select the project where you want to change the user permissions.
+2. Under **Settings**, click **Access**.
+3. Select a user and change the permissions.
+4. Click **Save**.
+5. After you delete a user from a project or an environment type, you must trigger a redeploy to propagate SSH access changes to each environment. You can redeploy by clicking **Redeploy** in the console.
+---
+title="Using the CLI"
+---
+Use the [Platform.sh CLI (Command Line Interface)](/development/cli/_index.md)to change user permissions.
 
-```bash
-$ platform get <projectID>
+* `platform user:role`
 
-```
+  ```bash
+  platform user:role user1@example.com -r contributor
+  ```
 
-or the `git clone` command visible from the "Git" dropdown in the management console
+Use `platform list` to get the full list of commands.
 
-```bash
-$ git clone git@github.com:user/github-repo.git Project Name
-```
+After you delete a user from a project or an environment type, you must trigger a redeploy to propagate SSH access changes to each environment. You can redeploy by using the CLI command platform `redeploy`.
```

Comment:
I would check if this is also done automatically. At least change it from being about deleting users.